x/btcstkconsumer/keeper/query.go

```diff
@@ -0,0 +1,7 @@
+package keeper
+
+import (
+	"github.com/babylonlabs-io/babylon/x/btcstkconsumer/types"
+)
+
+var _ types.QueryServer = Keeper{}
```

Comment:
we already have this in [grpc_query](https://github.com/babylonlabs-io/babylon/blob/ce31233233d496868bd1c83794c30ec3456c0b16/x/btcstkconsumer/keeper/grpc_query.go#L18)
File could be removed